services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -45,6 +47,31 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
         }
       }
     }
+
+    "getFileCommits" should {
+      val dummyCommitsJson: String = s"[${Json.stringify(Json.toJson(dummyCommit))}]"
```

Comment:
Almost the same problem as [here](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/63/files#r417570442)
I like that you didn't make it `lazy`.